### PR TITLE
Fix issue #77 Typescript compiler error 'TS1362: JobStatus can not be used as a value'

### PR DESCRIPTION
**Pull request by codeboyzhou**
Hi, I'm using `toad-scheduler` in my personal project, I saw the issue #77 when I was learning about this repository, I am interested in this problem and have a try to fix it.

Maybe there is a better way to avoid this compilation error via custom `type` instead of `enum`, like this `export type JobStatus = 'running' | 'stopped'`, and when using it, as follows:
```
if (job.getStatus() === 'running') {
  console.log('Job is running!');
}
```
Users don't need to `import JobStatus` explicitly, and the IDE (I'm using Microsoft VS Code) can also give the code auto-complete in a right way. If you used other values carelessly (neither `running` nor `stopped`) in your code, IDE will also give an error tips to remind you to correct it.

### CODE REVIEW

File: lib/common/Job.ts [hunk 1]
```diff
@@ -1,7 +1,4 @@
-export enum JobStatus {
-  RUNNING = 'running',
-  STOPPED = 'stopped',
-}
+export type JobStatus = 'running' | 'stopped'
```
Comment (kibertoad) [inline lib/common/Job.ts:1]: I'd prefer to support both approach, export enum and make it possible to pass value by string.
Can you adjust type of `JobStatus` to be values of JobStatusEnum, and export both?

Comment (codeboyzhou) [reply lib/common/Job.ts:1]: Sorry, I'm not sure I understood what you mean correctly, but could you view my second commit (e97f435)? Is it like that?

Comment (codeboyzhou) [reply lib/common/Job.ts:1]: @kibertoad Hi, are you here? :)

Comment (kibertoad) [reply lib/common/Job.ts:1]: yup, will do a little tweak and merge today

**Pull request merged**